tools/bin/commands/util/operator_func.bats

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
```

Comment:
What runs this test? Is it a part of the CI?

Reply:
I run it locally to test my stuff. CI could run it if we include bats in it.

Reply:
Please do, otherwise there's little value in it, no one will remember to run it and then we'll need to fix it after it was broken...

Reply:
I will do a separate PR for this